.openshift-ci/runners.py

```diff
@@ -82,6 +83,7 @@ def run(self):
                 hold = err
 
         if hold is not None:
+            print(f"Exception \"{str(hold)}\" raised in ClusterTestRunner", file=sys.stderr)
```

Comment:
Please check if it works without explicit `str()`.

Also, there's a second `if - raise` block below where a similar logging makes sense.

Please check if printing is here necessary. Normally, if exception is raised and not caught, it should be printed by the Python runtime with a stack trace. If it's not printed in our case, there's some code catching the exception that doesn't print its full detail.

Reply:
Turns out printing here is not necessary, I was using it for debug purposes but now it can go away. Instead I'm catching this outside of the testrunner and printing something myself.